src/lesson6/Movie.java

```diff
@@ -0,0 +1,60 @@
+package lesson6;
+
+import java.io.Console;
+import java.util.logging.XMLFormatter;
+
+public class Movie {
+
+    String title;
+    boolean [][] places;
+    int rows;
+    int seats;
+
+    Movie(String movieTitle, int rows, int seats){
+        this.title = movieTitle;
+        this.rows = rows;
+        this.seats = seats;
+        places = new boolean[rows][seats];
+        for(int i=0; i<rows; i++){
+            for (int j = 0; j < seats; j++) {
+                places[i][j] = true;
+            }
+        }
+    }
+
+    public String getTitle(){
+        return title;
+    }
+
+    public boolean ReserveSeat(int row, int seat){
```

Comment:
I detect that this code is problematic. According to the [Bad practice (BAD_PRACTICE)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#bad-practice-bad-practice), [Nm: Method names should start with a lower case letter (NM_METHOD_NAMING_CONVENTION)](https://spotbugs.readthedocs.io/en/stable/bugDescriptions.html#nm-method-names-should-start-with-a-lower-case-letter-nm-method-naming-convention).
Methods should be verbs, in mixed case with the first letter lowercase, with the first letter of each internal word capitalized.